xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAdapters.java

```diff
@@ -350,4 +349,37 @@ public static OpenPositions adaptOpenPositions(List<FtxPositionDto> ftxPositionD
   public static BigDecimal lendingRounding(BigDecimal value) {
     return value.setScale(4, RoundingMode.DOWN);
   }
+
+  public static Ticker adaptTicker(FtxResponse<FtxMarketDto> ftxMarketResp,
+                                   FtxResponse<List<FtxCandleDto>> ftxCandlesResp,
+                                   CurrencyPair currencyPair) {
+
+    System.out.println("marketResp"  + ftxMarketResp.toString());
```

Comment:
just saw this, these should be done using the logger framework not system.println